bundles/org.smarthomej.transform.basicprofiles/README.md

```diff
@@ -177,3 +177,32 @@ Switch motionSensorFirstFloor {
     channel="deconz:colortemperaturelight:AAA:BBB:brightness" [profile="basic-profiles:time-range-command", inRangeValue=100, outOfRangeValue=15, start="08:00", end="23:00", restoreValue="PREVIOUS"]
 }
 ```
+
+## State Filter Profile
+
+This filter passes on state updates from a (binding) handler to the item if and only if all listed item state conditions
+are met (conditions are ANDed together).
+Option to instead pass different state update in case the conditions are not met.
+State values may be quoted to treat as `StringType`.
+
+Use case: Ignore values from a binding unless some other item(s) have a specific state.
+
+### Configuration
+
+| Configuration Parameter | Type | Description                                                                                                                                                               |
+|-------------------------|------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `conditions`            | text | Comma separated list of expressions on the format `ITEM_NAME OPERATOR ITEM_STATE`, ie `MyItem EQ OFF`. Use quotes around `ITEM_STATE` to treat value as string ie `'OFF`' |
+| `mismatchState`         | text | Optional state to pass instead if conditions are NOT met. Use quotes to treat as `StringType`                                                                             |
```

Comment:
Why is it necessary to quote strings? There is also an issue with the way you split the string. If the comparison value contains a `,`, you split in the middle of a string. I know that this is really hard to handle correctly.

Reply:
Example why it may be useful (required) to quote strings

I have an aircon with a `mode` (string) channel. It takes values such as `HEAT`, `COOL`, `OFF`and so on. In order to not re-create state parsing, I've used the OH `TypeParser`. For a value `OFF` it will return `OnOffType.OFF`, and not the string `OFF`.

I guess I could improve further by checking the item type, but that would also make the same syntax mean 2 different things, and my gut feeling is that there will still be situations where the (hypotetical) user would want the non-default functionality.

Regarding comma as separator between multiple expressions; made this configurable.